src/app/styles/_line-truncation.scss

```diff
@@ -0,0 +1,22 @@
+// Styles extracted from here: https://css-tricks.com/line-clampin/
+@mixin line-truncation($line-height, $max-lines, $background-color: white) {
+  position: relative;
+  line-height: $line-height;
+  height: calc(#{$line-height} * #{$max-lines});
+  overflow: hidden;
+
+  &:after {
+    content: "";
+    text-align: right;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    width: 70%;
+    height: $line-height;
+    background: linear-gradient(
+      to right,
+      rgba(255, 255, 255, 0),
+      $background-color 50%
+    );
```

Comment:
one thing that bugged me was that the gradient fade out was too wide and happened in the middle of the line. I'd like to see it fade out right on the edge